recipes/unifhy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "unifhy" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/unifhy-org/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d219ad33ea94f78aaddc9757fadc0793f008dee33d5ebbca4e32c0b42fbf9170
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
This can probably be noarch. The lack of esmpy on Windows is fine and it is better to make it noarch than hammering the CIs.

Reply:
Thank you for the advice @ocefpaf, I have turned the build into noarch now so it should only be built once now. :-)